ts/state/ducks/search.ts

```diff
@@ -170,23 +168,46 @@ function startNewConversation(
 
 async function queryConversationsAndContacts(
   providedQuery: string,
-  options: { ourNumber: string; noteToSelf: string }
+  options: SearchOptions
 ) {
-  const { ourNumber, noteToSelf } = options;
+  const { ourNumber, noteToSelf, isSecondaryDevice } = options;
   const query = providedQuery.replace(/[+-.()]*/g, '');
 
   const searchResults: Array<ConversationType> = await searchConversations(
     query
   );
 
+  const ourPrimaryDevice = isSecondaryDevice
+    ? await getPrimaryDeviceFor(ourNumber)
+    : ourNumber;
+
+  const resultPrimaryDevices: Array<string | null> = await Promise.all(
+    searchResults.map(
+      async conversation =>
```

Comment:
unneeded `async`

Reply:
I added async because eslint gave me an error saying 
`Functions that return promises must be async`...
[c.f. docs](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/promise-function-async.md)
AFAIK I had two options:
- make the anonymous function `async` and use `Promise.resolve()` to be on par with `getPrimaryDeviceFor` which also returns a promise.
- remove the `async` and remove the `Promise.resolve()`, which seems against the eslint rule.

Reply:
Oh i didn't know eslint complained like that. That's a fair point then